### DISCUSSION TITLE
how to include/exclude spesific metrics

### DISCUSSION THREAD

**Discussion by sholker** [kbudde/rabbitmq_exporter#283] [Q&A] [answered]
Hi,
Im using rabbitmq_exporter . I would like to include just 'rabbitmq_queue_messages' metric.
I tried to using include_metrics and exclude_metrics(opposite). but I get the same metric with no changes.

```
   rabbitmq_prometheus,[
         {
          tcp_config,[
             {ip, "0.0.0.0"},{port, 15692}
          ]
         },
             {include_metrics, "rabbitmq_queue_messages"},
             {return_per_object_metrics, true},
       ]
     },
```

thanks

**Comment by kbudde** [comment, marked as answer]
Hi,

The answer is simple 😁

Your configuration looks like you want to configure the rabbitMQ exporter Plugin.

This repository here provides an external exporter for RabbitMQ. It was developed before there was an official plugin but it is still working in recent rabbitMQ versions. So, if you want to use it feel free 

But please checkout the official rabbitMQ documentation for the plugin.

BR

Kris
